<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProperties</class>
 <widget class="QDialog" name="DialogProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Delete Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCommand">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxCommands">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2" colspan="3">
      <widget class="QComboBox" name="comboBoxModels">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelModel">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPropertyLocation">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxPropertyLocations">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelPropertyName_1">
       <property name="text">
        <string>Property 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxPropertyNames_1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelPropertyName_2">
       <property name="text">
        <string>Property 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxPropertyNames_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelPropertyName_3">
       <property name="text">
        <string>Property 3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxPropertyNames_3">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelNewPropertyName">
       <property name="text">
        <string>New name</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditNewPropertyName">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="comboBoxSourceType">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="labelPropertyTo">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QComboBox" name="comboBoxTargetType">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelDataType">
       <property name="text">
        <string>Data type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxCommands</tabstop>
  <tabstop>comboBoxModels</tabstop>
  <tabstop>comboBoxPropertyLocations</tabstop>
  <tabstop>comboBoxPropertyNames_1</tabstop>
  <tabstop>comboBoxPropertyNames_2</tabstop>
  <tabstop>comboBoxPropertyNames_3</tabstop>
  <tabstop>lineEditNewPropertyName</tabstop>
  <tabstop>comboBoxSourceType</tabstop>
  <tabstop>comboBoxTargetType</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
